rasa/cli/arguments/data.py

```diff
@@ -43,3 +45,9 @@ def set_split_arguments(parser: argparse.ArgumentParser):
         default="train_test_split",
         help_text="Directory where the split files should be stored.",
     )
+
+
+def set_validator_arguments(parser: argparse.ArgumentParser):
+    add_domain_param(parser)
+    add_nlu_data_param(parser, help_text="File or folder containing your NLU data.")
+    add_stories_param(parser)
```

Comment:
Any chance we could use the `--data` path to all data, instead of `--stories` and `--nlu` in the same way `rasa train` does? Ideally it would get the directory for using in the same way, i.e. rasa train automatically pulls NLU data and story files while ignoring all other files. Currently I get an error if I try to use `rasa data validate` (ideal easiest usage) with no arguments because there is an extra file in my data folder.